tests/integration_tests/test_fee_history.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+from web3 import Web3
+
+from .network import setup_ethermint
+from .utils import (
+    ADDRS,
+    send_transaction,
+)
```

Comment:
Nit: apparently flake8 wants this sorted differently
```suggestion
from .utils import send_transaction, ADDRS
```